### PR TITLE
lesson-1 initial material in README

### PR DESCRIPTION
**Pull request by Aprillion**
await 17:00 next version for code review of examples

won't be able to teach 4 different examples in 2 hours if people are supposed to work on it themselves, let's focus on 1?

### CODE REVIEW

**Comment by lspdv** [comment]
ok its ok! 1 example is perfect if we let people to work themselves little bit more :-)

File: lesson-1/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+* http://caniuse.com/ provides a way to check feature support as well as a list of "modern" browsers
+
+_Exercise #1:_
+* download and open [jquery/v1.html](jquery/v1.html)
+* edit to remove jQuery dependency, targeting IE11+ (assuming last 2 and "extended support" versions of other browsers)
+
+
+## MV* frameworks
+
+Comparison of simple-but-practical implemantation of the same TODO app in various MV* frameworks, libraries
```
Comment (PavelVanecek) [inline lesson-1/README.md]: Typo _ implemantation_

File: lesson-1/react/current/.gitignore [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```
Comment (PavelVanecek) [inline lesson-1/react/current/.gitignore]: + Thumbs.db

+ .idea, .vscode?

**Comment by Aprillion** [comment]
I will be in auditorium at 17:00 if we need to make changes in my exercise solutions.

See next PR :)

**Pull request merged**